theme/src/components/container/_container_content.scss

```diff
@@ -1,3 +1,9 @@
 .homepage-content-container {
   margin-top: 2.5em;
 }
+
+@media (max-width: 767px) {
```

Comment:
You might want to use a variable here eg breakpoint-mobile or breakpoint-small since `767px` is also used in other styles.

Reply:
I have created the variable `$breakpoint-mobile` and implemented it throughout the page